<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamElementsReportIssueDialog</class>
 <widget class="QDialog" name="StreamElementsReportIssueDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreamElements.ReportIssue.Title</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>StreamElements.ReportIssue.Issue.Label</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="buddy">
    <cstring>txtIssue</cstring>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtIssue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>591</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkCollectLogsAndSettings">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>StreamElements.ReportIssue.SendLogsAndSettings.Label</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>StreamElements.ReportIssue.Action.Send</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>StreamElements.ReportIssue.Action.Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdOK</sender>
   <signal>clicked()</signal>
   <receiver>StreamElementsReportIssueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>StreamElementsReportIssueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtIssue</sender>
   <signal>textChanged()</signal>
   <receiver>StreamElementsReportIssueDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
